app/assets/stylesheets/views/_taxons.scss

```diff
@@ -243,14 +242,4 @@
       font-weight: bold;
     }
   }
-
-  .subsection-content {
-    display: none;
-  }
-
-  .subsection-is-open {
-    .subsection-content {
-      display: block;
```

Comment:
As discussed, this is still a `block`, as I incorrectly was using this to show/hide content (rather than set the display property).